### PR TITLE
Python3 linkage

### PR DESCRIPTION
**Pull request by sgn**
#### Testing the changes
- I tested the changes in this PR: **YES**|**briefly**|**NO**

### CODE REVIEW

File: common/hooks/pre-configure/02-script-wrapper.sh [hunk 1]
```diff
@@ -65,17 +65,27 @@ _EOF
...
 --includes|--cflags)
 	echo "-I${XBPS_CROSS_BASE}/usr/include/python${version}" ;;
+_EOF
+	if [ "$shlib" ]; then
+		cat <<_EOF
+--ldflags)
+	echo "-lpython${shlib}" ;;
+_EOF
+	fi
+	cat <<_EOF
```
Comment (ahesford) [inline common/hooks/pre-configure/02-script-wrapper.sh:84]: I'm surprised we haven't needed to wrap more of `python-config` already!

**Review by ahesford — APPROVED** [review]
I like this approach. It helps us to make sure everything that needs the Python shared lib is available but doesn't force linking on everything that might need to include `python3-devel` for some other reason.

**Comment by sgn** [comment]
Abandon this idea, too many things to think about.

**Comment by eli-schwartz** [comment]
Fixing mod_wsgi to link to libpython is correct, because upstream's configure script does so already. It was just an oversight that had it use python-config without `--embed`.

Generally linking to libpython for extension modules is probably wrong -- upstream Python specifically moved to requiring --embed for this, because anything that runs inside the python interpreter is expected to load whichever library the interpreter provides.

**Pull request closed without merging**